src/modules/_partials/shared/Paginations/index.tsx

```diff
@@ -1,94 +1,99 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
+import React, { useState } from 'react';
+
+interface PaginationProps {
+  users: any[];
```

Comment:
I think you can rename this to `items` as it can be used somewhere else not only `users`